tabwriter/tabwriter.go

```diff
@@ -283,12 +285,24 @@ func (b *Writer) writeLines(pos0 int, line0, line1 int) (pos int) {
 			} else {
 				// non-empty cell
 				useTabs = false
-				if b.flags&AlignRight == 0 { // align left
+				alignRight := b.alignment[j] == AlignRight
```

Comment:
What if you want the table aligned right and override some columns to be aligned left?